common/formatting/layout_optimizer_internal.h

```diff
@@ -201,6 +201,14 @@ using LayoutTree = VectorTree<LayoutItem>;
 // Single segment of LayoutFunction
 // Maps starting column to a linear cost function and its optimal layout.
 struct LayoutFunctionSegment {
+  LayoutFunctionSegment(int column, const LayoutTree& layout, int span,
```

Comment:
If we just remove he constructors below, we should not need this workaround (the constructors below are just default copy and move)